php/Terminus/Models/Environment.php

```diff
@@ -648,9 +648,23 @@ public function sendCommandViaSsh($command)
             'ssh -T %s@%s -p %s -o "AddressFamily inet" %s',
             [$sftp['username'], $sftp['host'], $sftp['port'], escapeshellarg($command),]
         );
+
+        // Catch Terminus running in test mode
+        $test_mode = isset($_ENV["TERMINUS_TEST_MODE"])
+            && filter_var($_ENV["TERMINUS_TEST_MODE"], FILTER_VALIDATE_BOOLEAN);
+        if ($test_mode) {
```

Comment:
We can't use Behat's fixtures for anything outside HTTP API requests so I've trapped the SSH calls here.
Is there a better PHP idiom for testing an ENV var... or a better approach?